src/select.tsx

```diff
@@ -385,7 +386,19 @@ const Select: React.FC<SelectProps> = ({
 
                 <div
                     className={styles.selectTextVariants[disabled ? 'disabled' : 'default']}
-                    style={{top: label ? 27 : 17}}
+                    style={{
+                        top: isDesktopOrBigger
+                            ? label
+                                ? 28
```

Comment:
Are we sure this is necessary? I don't understand the logic behind all these numbers

Reply:
Try something like this: [Playroom](https://mistica-web.vercel.app/playroom#?code=N4Igxg9gJgpiBcIA8UCWA3AOgOwAS4GcAXATwBsYBeYYHffABwgNSNQm3l0xACcYyAQzboYPADR16Ad1RQiACy48AzAAY1DAB4Sp%2BBTFQBzBUWUgArBu2689AEaCwAayO8IAV2xRz-KLfwAX0CcAD4pFAw9QlIKalo7elwmFjYOc0F7AggyDyIxEElEmTlFcwBGax1C6P1DEzNuEErNaqKk-EcXN09vc3shFwCOoggGLjV2jvsIIlGAWwmppPnBXiNUbAAVMYy8iGGVtY3sACFZhb3Rw9xgqQB6cOwke7R0J8KQRRh5mAIEADaIFOQg8AH1UAB5ADKIAAuuIQLJ5Ap-vAASoAExqOGBIA)

Reply:
yes, please, can you explain all this logic? why do we need different styles for Safari?

Reply:
insane pixel perfect 😅
No safari logic
![image](https://github.com/Telefonica/mistica-web/assets/6722153/8df88c26-45c8-47b2-8a03-302cd8746b6a)

Safari logic
![image](https://github.com/Telefonica/mistica-web/assets/6722153/bfd9e65f-4d31-4135-960d-713e5201e553)

It is true that with the removal of inherit fontsize for input fields maybe it is not worth this pixel perfect. I can remove it if you consider it too much, no problem.

Reply:
I'm still not sure why Safari has differences. I believe the same results can be achieved with pure CSS. Maybe we could leave this logic as it is, but also create a ticket to see if we can handle it without ifs.

What do you think @atabel ?